frontend/src/app/main/data/workspace/libraries_helpers.cljs

```diff
@@ -1354,9 +1379,51 @@
                      (conj roperations roperation)
                      (conj uoperations uoperation)))))))))
 
+(defn- propagate-attrs
+  "Helper that puts the src-shape attributes (attrs) into tgt-shape but only if
+  not touched the group or if omit-touched? flag is true"
+  [tgt-shape src-shape attrs omit-touched?]
```

Comment:
I suggest to use the "origin" and "dest" terms, to be consistent with other functions in the module.

Reply:
OK